ci-operator/step-registry/ipi/conf/azure/techpreview/ipi-conf-azure-techpreview-chain.metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+	"path": "ipi/conf/azure/techpreview/ipi-conf-azure-techpreview-chain.yaml",
+	"owners": {
+		"approvers": [
+			"abhinavdahiya"
```

Comment:
I guess inherited from `master`, but ideally each directory will have more than one approver.  Can we find anyone else who can sign up to help maintain this chain going forward?